src/app/modules/group/components/pending-request/pending-request.component.ts

```diff
@@ -143,20 +145,31 @@ export class PendingRequestComponent implements OnInit, OnChanges {
     });
   }
 
+  processRequests(action: Action) {
+    const requestMap = new Map<string, string[]>();
+    this.selection.forEach(elm => {
+      const groupID = elm.group.id;
+      const memberID = elm.user.group_id;
+
+      const value = requestMap.get(groupID);
+      if (value) requestMap.set(groupID, value.concat([memberID]));
+      else       requestMap.set(groupID, [memberID]);
```

Comment:
We don't space indent in the middle of the line to align in this project.
```suggestion
      else requestMap.set(groupID, [memberID]);
```